_locales/en/messages.json

```diff
@@ -19,16 +19,20 @@
       "message": "&Help",
       "description": "The label that is used for the Help menu in the program main menu. The '&' indicates that the following letter will be used as the keyboard 'shortcut letter' for accessing the menu with the Alt-<letter> combination."
     },
+    "mainMenuSettings": {
+      "message": "Preferences…",
+      "description": "The label that is used for the Preferences menu in the program main menu. This should be consistent with the standard naming for ‘Preferences’ on the operating system."
+    },
     "menuSetupWithImport": {
-      "message": "Set up with import",
+      "message": "Set Up with Import",
```

Comment:
Consistency is good, and I support this change. But it should be noted that on Windows, sentence case is preferred while on OSX capitalization like this is preferred.

Reply:
Thanks for pointing this out. I checked some Windows apps and ones with a classic menu bar like ours (Notepad, Visual Studio, Visual Studio Code, Slack, Sublime Text) use _Title Case_ whereas apps with new style UI (Edge, WordPad, etc.) use _Sentence case_, although sometimes inconsistently. I just opted for consistency here as the strings above stood out. Hope that’s cool 😄